### PR TITLE
Wiki rework

### PR DESCRIPTION
**Pull request by dakyskye**
- added wiki storage functions in the storage's main, Stoner interface
- as of the first change, replaced using external storage module with local function variables
- changed wiki pulling from being independent ticker to a bot's heartbeat

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -37,5 +37,5 @@ log.txt
 cj
 
 # directory
-wiki/
+/wiki/
```
Comment (Southclaws) [inline .gitignore]: This is a root path, not sure what it means in gitignore tbh, why was this change necessary?

File: storage/storage.go [hunk 2]
```diff
@@ -36,6 +36,9 @@ type Storer interface {
 	GetReadmeMessage() (message string, err error)
 	FetchReadmeMessage(gistID string, gistFile github.GistFilename) (message string, err error)
 	UpdateReadmeMessage(session *discordgo.Session, original *discordgo.Message, upstream string) (err error)
+
+	PullWiki(wikiURL string) (err error)
```
Comment (Southclaws) [inline storage/storage.go]: Since this storer interface is only used for database stuff, I think it would make more sense to create a _new_ interface specifically just for the wiki - this separates the concerns in a clearer way and doesn't add a MongoDB dependency to the wiki implementation.

File: .gitignore (hunk 1 above)
Comment (dakyskye) [reply .gitignore]: Because I added wiki folder in heartbeat which was getting ignored, so I had to change this to root path only as wiki git repository is cloned only here.

File: storage/storage.go (hunk 2 above)
Comment (dakyskye) [reply storage/storage.go]: Oh I see 👍 I'll make new interface for that. I guess I've to remove it from [Memory](https://github.com/dakyskye/cj/blob/wiki-rework/storage/memory.go#L68-L73) interface because it seems it's used to fake database related functions.

File: .gitignore (hunk 1 above)
Comment (Southclaws) [reply .gitignore]: ahhh I understand now!
